### PR TITLE
feat: Change saving indicator wordings

### PR DESCRIPTION
**Pull request by edas**
These wordings are for the saving indicator in the bottom right in large views (not displayed yet in small views). Thenew wordings are the one requested by @joel-costa for a better UX

### CODE REVIEW

File: src/locales/fr.json [hunk 1]
```diff
@@ -31,16 +31,16 @@
...
-      "saved_day_ago": "Enregistré il y a %{time} jour.",
-      "saved_days_ago": "Enregistré il y a %{time} jours.",
+      "saved_just_now": "Dernière mise à jour à l'instant.",
+      "saved_secs_ago": "Dernière mise à jour il y a %{time} sec",
+      "saved_min_ago": "Dernière mise à jour il y a %{time} min",
+      "saved_mins_ago": "Dernière mise à jour il y a %{time} min",
+      "saved_hour_ago": "Dernière mise à jour il y a %{time} heure",
+      "saved_hours_ago": "Dernière mise à jour il y a %{time} heures",
+      "saved_day_ago": "Dernière mise à jour il y a %{time} jour",
+      "saved_days_ago": "Dernière mise à jour il y a %{time} jours",
```
Comment (Crash--) [inline src/locales/fr.json]: Does it fit well on a very small screen?

Comment (edas) [reply src/locales/fr.json]: We don't display this on small screens yet (we will, but the question will be raised at this time when I will ask for the correct UI for small screens))

File: src/locales/en.json [hunk 2]
```diff
@@ -31,16 +31,16 @@
...
     "SavingIndicator": {
-      "saved_just_now": "Saved just now.",
-      "saved_secs_ago": "Saved %{time} sec ago",
-      "saved_min_ago": "Saved %{time} min ago",
-      "saved_mins_ago": "Saved %{time} min ago",
-      "saved_hour_ago": "Saved %{time} hour ago",
-      "saved_hours_ago": "Saved %{time} hours ago",
-      "saved_day_ago": "Saved %{time} day ago",
-      "saved_days_ago": "Saved %{time} days ago",
+      "saved_just_now": "Last update now",
```
Comment (y-lohse) [inline src/locales/en.json]: Shouldn't it be `Last updated`? Or `Last update: now` or `last update was now`, but `Last update now` seems wrong.

File: src/locales/en.json [hunk 3]
```diff
@@ -31,16 +31,16 @@
...
+      "saved_secs_ago": "Last update %{time} sec. ago",
+      "saved_min_ago": "Last update %{time} min. ago",
+      "saved_mins_ago": "Last update %{time} min. ago",
+      "saved_hour_ago": "Last update %{time} hour ago",
+      "saved_hours_ago": "Last update %{time} hours ago",
+      "saved_day_ago": "Last update %{time} day ago",
+      "saved_days_ago": "Last update %{time} days ago",
       "saving": "Saving...",
-      "saving_impossible": "Impossible to save.",
+      "saving_impossible": "Saving... ...",
```
Comment (y-lohse) [inline src/locales/en.json]:
```suggestion
      "saving_impossible": "Saving...",
```

File: src/locales/fr.json [hunk 4]
```diff
@@ -31,16 +31,16 @@
...
       "saving": "Enregistrement en cours...",
-      "saving_impossible": "Enregistrement impossible.",
+      "saving_impossible": "Enregistrement en cours... ...",
```
Comment (y-lohse) [inline src/locales/fr.json]:
```suggestion
      "saving_impossible": "Enregistrement en cours...",
```

File: src/locales/en.json (hunk 2 above)
Comment (edas) [reply src/locales/en.json]: not as it is requested : https://trello.com/c/T1FZP8U9/132-etu-jai-linformation-de-lenregistrement-de-ma-note-2j

File: src/locales/en.json (hunk 3 above)
Comment (edas) [reply src/locales/en.json]: nope, @joel-costa requires explicit 2 times 3 dots

File: src/locales/fr.json (hunk 4 above)
Comment (edas) [reply src/locales/fr.json]: nope, @joel-costa requires explicit 2 times 3 dots

File: src/locales/en.json (hunk 3 above)
Comment (y-lohse) [reply src/locales/en.json]: Mmh ok, but that is super strange, and IMO not clear at all. At best it makes it look like we're trying to save again which is not what we do AFAIK?

I'm not sure if it's intentional or not, but in the trello card there is no space between the sets of dots, it more 6 dots than 2 sets of 3 dots.

**Comment by edas** [comment]
Vu que la carte a pris du temps, j'ai mis à jour avec l'étape supplémentaire (ajout de l'âge de la dernière sauvegarde quand une sauvegarde est en attente depuis longtemps)

File: src/lib/utils.spec.js [hunk 5]
```diff
@@ -137,3 +138,27 @@ describe('getFolderLink', () => {
     })
   })
 })
+
+describe('relativeAge', () => {
+  // constants
+  const sec = 1000
+  const min = 60 * sec
+  it('should have the recorded response for age=3s', () => {
```
Comment (ptbrowne) [inline src/lib/utils.spec.js]: Nit: "should return the right data for age=3s" is a bit easier to understand (I couldn't understand why "recorded" was used, I looked for a memoize in the implementation)

File: src/lib/utils.js [hunk 6]
```diff
@@ -153,3 +153,64 @@ export function createNoteDocument(client) {
...
+]
+
+/**
+ * @typedef {object} RelativeAge
+ * @property {string} key - translation (sub) key
+ * @property {unit} unit - unit (minute, second, hour…) in which the age will be displayed
+ * @property {integer} coef - in how much values of this unit do we count (like every 10 minutes)
+ * @property {integer} time - age to be inserted in the translation message
+ * @property {integer} interval - `unit * coef`: compute again in that timeframe (in milliseconds)
+ */
```
Comment (ptbrowne) [inline src/lib/utils.js]: :+1:

**Comment by edas** [comment]
New wordings from https://fabien-notes.mycozy.cloud/public/?sharecode=afSztF186uGS#/

**Pull request merged**